impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/customStereotype/MyService.java

```diff
@@ -1,5 +1,6 @@
 package org.jboss.cdi.tck.tests.build.compatible.extensions.customStereotype;
 
+@DummyStereotype
```

Comment:
Why is this required? The class already has a stereotype?

Reply:
Originally, `MyService` was registered via `@Discovery` manually, I simply made sure it will be replaced by natural discovery process.